airgun/entities/product.py

```diff
@@ -75,6 +77,53 @@ def synchronize(self, entity_name):
         view.progressbar.wait_for_result()
         return view.read()
 
+    def _select_action(self, action_name, entities_list, values=None):
+        """Navigate to all entities, select the entities, and returns the view
+        of the selected action name from main entity select action dropdown.
+        """
+        return self.navigate_to(
+            self,
+            'Select Action',
+            action_name=action_name,
+            entities_list=entities_list,
+            values=values
+        )
+
+    def apply_action(self, action_name, entities_list, values=None):
+        """Apply action name for product/products"""
+        view = self._select_action(action_name, entities_list, values=None)
+        if values['http_proxy_policy'] == "Global Default":
+            values['http_proxy_policy'] = view.http_proxy_policy.all_options[0][0]
+        view.fill(values)
+        view.update.click()
+        view.flash.assert_no_error()
+        view.flash.dismiss()
+
+    def manage_http_proxy(self, entities_list, values):
+        """Manage HTTP Proxy for product/products
+
+        :param entities_list: The product names to perform Manage HTTP Proxy action.
+        :param values: dict containing http_proxy_policy and http_proxy values.
+            eg: {'http_proxy_policy': 'No HTTP Proxy'}, {'http_proxy_policy': 'Global Default'},
+            {'http_proxy_policy': 'Use specific HTTP Proxy', 'http_proxy': 'proxy_name'}
+        """
+        view = self._select_action('Manage HTTP Proxy', entities_list, values=None)
+        if values['http_proxy_policy'] == "Global Default":
+            values['http_proxy_policy'] = view.http_proxy_policy.all_options[0][0]
+        view.fill(values)
+        view.update.click()
+        view.flash.assert_no_error()
+        view.flash.dismiss()
+
+    def sync_selected(self, entities_list):
```

Comment:
Do we need that option option at this moment? We already have `synchronize` method that syncs single product. How often do you plan to sync multiple products at once, and how many products?

Reply:
Nope, we don't need it right now. Removed `sync_selected` method.